### DISCUSSION TITLE
Timeline Item Badge background color in the issus and pull request page seems to be incorrect

### DISCUSSION THREAD

**Discussion by adaex** [community/community#5353] [Projects and Issues]
Example : https://github.com/github/docs/pull/9381

It seems that the color of the badge is lost and the first badge is a bit strange.

![image](https://user-images.githubusercontent.com/29560987/131313854-ac4b3743-9506-402a-a2a0-c0ebfcd4c861.png)

**Comment by Shinigami92** [comment]
I can reproduce this with the dark theme!

Compare the different greens:

<img src="https://user-images.githubusercontent.com/7195563/131379261-c697850a-c25c-4542-af89-c7d5b9157c55.jpg" width="100px"/>
